File: testsuite/file_io/tests/java_ac_case/post.java
import java.util.*;
import java.io.*;

public class post {
    public static void main(String args[]) throws IOException {
        BufferedReader cin = new BufferedReader(new FileReader("PoSt.InP"));
        PrintWriter cout = new PrintWriter(new FileWriter("PoSt.OuT"));
        String[] line = cin.readLine().split(" ");
        cout.println(Integer.parseInt(line[0]) + Integer.parseInt(line[1]));
        cin.close();
        cout.close();
    }
}

File: testsuite/file_io/tests/cpp_ac_iostream_case/post.cpp
#include <fstream>

using namespace std;

int main() {
  ifstream cin("PoSt.InP");
  ofstream cout("PoSt.OuT");
  int a, b;
  cin >> a >> b;
  cout << (a + b) << '\n';
  return 0;
}
